README.md

```diff
@@ -39,13 +41,16 @@ Note: If you intend to build from sources on Linux/macOS, follow [these instruct
 
 Before you build, you will need to have some additional software installed. This is the most straightforward path - see [Building Sources](https://github.com/Microsoft/DirectXShaderCompiler/wiki/Building-Sources) on the Wiki for more options, including Visual Studio 2015 and Ninja support.
```

Comment:
It needn't be now, but that wiki could probably use a similar update. It still advocates using -vs2017 with hctbuild and still suggests git and python need be installed separately from visual studio.

Somewhere it might make sense to mention that vs2019 can be used with the appropriate flag.